brainstormforce/astra
# AST-757 - Added black default color to editor component button in editor

### Description
- Need to follow WP's default black color to component-buttons in editor

### Screenshots
- 2020 theme - https://share.getcloudapp.com/4gund8ze (expected)
- Astra (till now) - https://share.getcloudapp.com/RBul8AP1
- In this PR - https://share.bsf.io/nOuvmQ20

### Types of changes
- Non-breaking change

### How has this been tested?
- Check editor view 
- Make sure if any other thing (outside editor area) not affected by this

### Checklist:
